pkg/storage/kube/build.go

```diff
@@ -65,11 +59,13 @@ func (s *store) CreateBuild(build *brigade.Build) error {
 			"payload": build.Payload,
 		},
 		StringData: map[string]string{
-			"project_id":     build.ProjectID,
-			"event_type":     build.Type,
-			"event_provider": build.Provider,
-			"commit":         build.Commit,
+			"build_id":       buildName,
 			"build_name":     buildName,
+			"commit_id":      build.Revision.Commit,
```

Comment:
I understand this, but on the other hand this make me think that we'd better name it `Commit` rather than `Revision`.

More concretely, I think if it looked like:

```
"commit_id": build.Commit.ID,
"commit_ref": build.Commit.Ref,
```

That's easier to read for me due to consistency(with env var naming. This is already mirrored to BRIGADE_COMMIT_ID and BRIGADE_COMMIT_REF anyways?) and less indirection hence less translation in my mental model of `Revision` to `Commit`.

Reply:
I have named this revision to keep it generic in case we add more fields (`base_ref`, `base_sha`, `pull_request`, `branch`, `head_remote_url`).  Also a build could be multiple commits.

Reply:
Thanks for the explanation!

I understood that you meant that a "revision"(and yeah, I got to know that git-scm.com does mention about the word "revision" many times) in Git can be represented by many ways like you've noted, right?

That makes sense to me.
However then, I'd prefer `build.Revision.SHA1` or maybe `build.Revision.SHA` to `build.Revision.Commit`.
If you name it `Commit`, I doubt if it conflicts with `base_sha`, which also refers to a specific "commit" by a sha1 hash.

Reply:
> Also a build could be multiple commits.

This is certainly what I had missed in my mind. Thanks 👍 

Probably we'd better have `revisions[].commit` rather than `revison.commits` in that case, though?

At least in git's terminology, a "revision" seems like representing a single state in the history, but not a "range between two states". If that's the case, IMHO, `revision.commits` sounds a bit unnatural.